samples/django/django_user/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==0.10.0
```

Comment:
yes thanks. manually wrote it out since my virtualenv has some unrelated stuff, hopefully would have caught this in final walkthrough. Thank you for taking look.